yarn.lock

```diff
@@ -1875,6 +1875,14 @@
     "@jridgewell/resolve-uri" "^3.1.0"
     "@jridgewell/sourcemap-codec" "^1.4.14"
 
+"@jridgewell/trace-mapping@^0.3.20":
```

Comment:
I'm a little surprised to see a new dependency here given that the [changelog](https://github.com/vitejs/vite/blob/v5.0.13/packages/vite/CHANGELOG.md#5013-2024-03-24) for 5.0.13 only has a [single change](https://github.com/vitejs/vite/commit/d2db33f) which doesn't introduce new dependencies.

Reply:
I think dependabot got lost and updated a bunch of other deps while updating vite. Maybe due to us using Yarn v1 and workspaces.

I tried to update Vite to 5.0.13 manually and it updates only related packages. But there are problems with `@esbuild` packages getting duped because electron-vite depends on esbuild ^0.19.8 and vite 5.0.13 depends on esbuild ^0.20.1.

This might cause problems (see #33356), so I'd have to test the changes first.